apps/storefront/pages/grunnleggende/for-designere/eget-tema.mdx

```diff
@@ -0,0 +1,101 @@
+**Alternativ 2:** _Med_ kobling mellom Figma og kode
+
+Hvilket alternativ du går for avhenger av dine behov og rammer.
+
+Fordelen med å ha en kobling, gjennom tokens, er at de sørger for at designere og utviklere arbeider etter de samme reglene, og at ting ser likt ut i Figma og i de faktiske løsningene. En annen fordel er at du vil kunne generere et tema som vil fungere ut av boksen med designsystemet sine komponentbibliotek i kode.
+
+Ulempen er at det krever en del mer første gang du skal sette det opp, og du er avhengig av pluginen [Tokens Studio](https://tokens.studio/). Du må vurdere om det er verdt det for din organisasjon. _Uten direkte kobling til tokens_, må du manuelt passe på at variablene som lages i Figma er de samme som brukes i kode.
+
+Vi guider deg gjennom begge alternativene her.
+
+## Alternativ 1: Uten kobling
+
+Å _ikke_ ha en kobling er den raskeste veien til å få opp Figma-filen med egne farger.
+
+### Slik går du frem for å få det til:
+
+1. Hent Designsystemet fra [Figma Community](https://www.figma.com/community/file/1322138390374166141/designsystemet-core-ui-kit) (Trykk "Open in Figma"). Merk at dette blir en kopi av komponentbiblioteket uten noen kobling mot hovedfilen.
+2. Filen har nå sannsyligvis lagt seg i dine "drafts" nå, så du må flytte den til et fornuftig sted i din organisasjon.
+3. Åpne "local variables" i Figma og bytt ut `accent`, `neutral`, `brand1`, `brand2`, `brand3` med dine egne farger. Har du ikke farger for hele skalaen, kan du [få hjelp med å sette det opp fargeskalaene her](/grunnleggende/for-designere/fargesystem).
+4. For at andre designere i organisasjonen skal kunne bruke komponentene må du publisere filen.
+5. Det var alt! Nå skal komponentene ha dine farger! 🎉 Og så var det å holde filen oppdatert da... Figma har foreløpig [ikke støtte for å hente ned endringer fra en Community-fil](https://forum.figma.com/t/need-a-way-to-get-latest-updates-or-subscribe-to-changes-from-a-published-community-file/32571). Vi håper dette vil komme, i mellomtiden kan du [vurdere disse alternativene](/grunnleggende/for-designere/hent-oppdateringer).
+
+<Image
+  src='/img/local-variables.png'
+  alt='Skjermbildet viser Figma med et utsnitt fra variablene'
+  boxShadow={false}
+  caption='Skjermbildet viser variabler i Figma. Du må bytte ut fargene under `alt1`, `alt2`, `alt3` med dine egne farger (Steg 3).'
+/>
+
+## Alternativ 2: Med kobling
+
+En kobling gjør at du slipper å legge variablene inn i Figma, du importerer dem i stedet fra json-filen og holder dem i sync via pluginen "Tokens studio".
+
+### Slik går du frem for å få det til:
+
+1. Er du designer kan det være lurt å snakke med en utvikler, slik at dere har en felles forståelse av hvordan tokens kobler sammen Figma og kode. For å kunne legge til egne tokens i dagens oppsett må du lage en kopi av dette repoet: [designsystemet-themes](https://github.com/designsystemet/design-tokens). Velg "Use this template" og "Create a new repository".
```

Comment:
Kunne vi lagt til en lenke til artikkelen som inneholder video av Mickey som viser hvordan det gjøres? På sikt kanskje lage en liten video som for å vise hvordan dette gjøres :)

Reply:
JA, men i den videoen vi har viser vi vel ikke hvordan du lager PAT. Så tror vi må lage en ny

Reply:
Jeg kan lage en video uten stemme evt. , der jeg bare går gjennom alt fra start til slutt? Da slipper vi å tekste, så får vi det ut litt fortere?